src/pudl_rmi/connect_deprish_to_ferc1.py

```diff
@@ -434,7 +434,7 @@ def execute(
         # Note: Right now we are just dropping the non-connected
         # not_connected = df_to_scale[df_to_scale.record_id_eia.isnull()]
         connected_to_scale = df_to_scale[~df_to_scale.record_id_eia.isnull()]
-
+        # does this do anything?
         df_to_scale = make_plant_parts_eia.reassign_id_ownership_dupes(df_to_scale)
```

Comment:
`df_to_scale` isn't used after this so i think this is leftover and should be taken out? In `aggregate_duplicate_eia` the first thing that is done is `make_plant_parts_eia.reassign_id_ownership_dupes(connected_to_scale)` which is what I think this line used to do.